var/spack/repos/builtin/packages/armpl-gcc/package.py

```diff
@@ -200,24 +200,33 @@ def install(self, spec, prefix):
         exe = Executable("./arm-performance-libraries_{0}_{1}.sh".format(armpl_version, get_os()))
         exe("--accept", "--force", "--install-to", prefix)
 
+    @property
+    def is_shared(self):
+        return self.spec.satisfies("+shared")
+
+    @property
+    def is_ilp64(self):
+        return self.spec.satisfies("+ilp64")
+
+    @property
+    def is_omp(self):
+        return self.spec.satisfies("threads=openmp")
```

Comment:
Instead of adding these functions, that are just shorthands to `self.spec.satisfies` with a specific argument, did you consider adding a `libname` and a `suffix` property to the package?

Reply:
Thank you for the idea. Done.